pre-docs/bolt_installing_modules.md

```diff
@@ -53,18 +59,18 @@ For more details about specifying modules in a Puppetfile, see the [Puppetfile d
 
      ```
      # Modules from the Puppet Forge.
-     mod 'puppetlabs/package', '0.2.0'
```

Comment:
@m0dular pointed out that these references were out of date. I think changing the example to modules not bundled with bolt will avoid users potentially installing different (older) versions of bundled modules.